docs/src/cox.md

```diff
@@ -0,0 +1,27 @@
+# Cox Proportional Hazards Model
+
+The [Cox proportional hazards model](https://en.wikipedia.org/wiki/Proportional_hazards_model) is a semiparametric regression used to fit survival models without knowing the distribution. It is based on the assumption that different covariates affect the hazard function multiplicatively:
+
+```math
+\lambda(t|X_i) = \lambda_0(t)\exp(X_i\cdot\beta)
+```
+
+where $\lambda(t|X_i)$ is the estimated hazard for sample $i$, $\lambda_0$ is the baseline hazard, $X_i$ the vector of covariates for sample $i$ and $\beta$ are the coefficients of the regression.
```

Comment:
Oh, Documenter doesn't use `mkdocs`. On the command line, `cd`'d into the Survival.jl directory, you can do
```
julia -e 'include(joinpath("src", "Survival.jl")); include(joinpath("docs", "make.jl"))'
```
Just make sure you've done `Pkg.add("Documenter")` from within Julia first.